### PR TITLE
Issue #2807601 by kim.pepper: Split out aGov base and whitlam themes

### PR DESCRIPTION
**Pull request by kimpepper**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ var compiledTheme = '/app/profiles/agov/themes/agov/agov_whitlam/';
 options.rootPath = {
   project     : __dirname + '/',
   web         : __dirname + '/app/',
-  baseTheme   : __dirname + '/app/profiles/agov/themes/agov/agov_base/',
+  baseTheme   : __dirname + '/app/profiles/agov/themes/contrib/agov_base/',
```
Comment (Sam152) [inline gulpfile.js]: Shouldn't the build files to compile each of these themes be in the theme repos themselves?

**Comment by Sam152** [comment]
Rest LGTM

File: gulpfile.js (hunk 1 above)
Comment (kimpepper) [inline gulpfile.js]: OK. Doing this over in https://github.com/previousnext/agov_base/pull/1

Comment (kimpepper) [inline gulpfile.js]: ..and here https://github.com/previousnext/agov_whitlam/pull/1

**Comment by rikki-iki** [comment]
test this please

**Comment by benjy** [comment]
Lets merge 8.x-1.x into your branch, could be behind?

**Comment by rikki-iki** [comment]
Rerolled, ready for NPM fix :)

**Pull request merged**